daemon/mgr/image.go

```diff
@@ -30,7 +30,9 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-var deadlineLoadImagesAtBootup = time.Second * 10
+// NOTE: make it to 10 minutes because it will take long time to load the
```

Comment:
Actually this is not overtime because of image, but the machine env, like load, or other thing, would you mind to change the comment? @fuweid

Reply:
yeah. it is one of reason. the biggest issue is high load in machine.

Reply:
Actually not vary accurately， also ok